frontend/src/components/project/Buttons/ContactCreatorButton.js

```diff
@@ -99,85 +98,52 @@ export default function ContactCreatorButton({
     : texts.responsible_person_project;
   const buttonText = texts.contact;
 
-  if (small) {
-    return (
-      <Button
-        variant="contained"
-        color="primary"
-        startIcon={<SendIcon />}
-        endIcon={<Avatar src={creatorImageURL} className={classes.smallAvatar} />}
-        onClick={handleClickContact}
-        className={className}
-      >
-        {buttonText}
-      </Button>
-    );
-  } else if (tiny) {
-    return (
-      <Button
-        variant="contained"
-        color="primary"
-        onClick={handleClickContact}
-        className={className}
-      >
-        {buttonText}
-      </Button>
-    );
-  } else if (large) {
-    return (
-      <div className={`${classes.largeButton} ${className}`} onClick={handleClickContact}>
-        <DetailledContactCreatorInfo
-          creatorName={creatorName}
-          creatorImageURL={creatorImageURL}
-          creatorsRoleInProject={creatorsRoleInProject}
-        />
-        <Button variant="contained" color="primary">
-          {buttonText}
-        </Button>
-      </div>
-    );
-  } else {
-    return (
-      <div
-        className={`${classes.root} ${className}`}
-        onMouseEnter={handleMouseEnter}
-        onMouseLeave={handleMouseLeave}
-        onClick={handleClickContact}
-      >
-        <div className={classes.buttonWithCollapseContainer}>
-          <Collapse in={hoveringButton} timeout={550}>
+  return (
+    <div
+      className={withInfoCard ? `${classes.root} ${className}` : null}
+      onMouseEnter={handleMouseEnter}
+      onMouseLeave={handleMouseLeave}
+      onClick={handleClickContact}
+    >
+      <div className={collapsable ? classes.collapsableContainer : classes.nonCollapsableContainer}>
```

Comment:
`nonCollapsableContainer` is a subset of `collapsableContainer`. Maybe I would extract this 'width' property to a separate class, which is always present, and add the  `collapsableContainer` as an optional second class (if required)

Reply:
Not sure if what I did is what you meant, see [commit](https://github.com/climateconnect/climateconnect/pull/859/commits/552213ec8dd98fce42c8f7236afc09643167ccf5)